CMakeLists.txt

```diff
@@ -229,8 +233,10 @@ if(HAVE_ARM64_CRC32C)
   endif(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
 endif(HAVE_ARM64_CRC32C)
 # CMake only enables PIC by default in SHARED and MODULE targets.
-set_property(TARGET crc32c_arm64
-    PROPERTY POSITION_INDEPENDENT_CODE "${BUILD_SHARED_LIBS}")
+if( BUILD_SHARED_LIBS OR CMAKE_POSITION_INDEPENDENT_CODE )
+  set_property(TARGET crc32c_arm64
+    PROPERTY POSITION_INDEPENDENT_CODE ON )
```

Comment:
This property will be set automatically.

Reply:
The PIC property is not automatically set for OBJECT targets (which is the case for the crc32c_arm64 and crc32_sse42) so it needs to be explicitly set.

The code before the pull request was enabling PIC if BUILD_SHARED_LIBS was true and disabling PIC if BUILD_SHARED_LIBS was false regardless of CMAKE_POSITION_INDEPENDENT_CODE being set